src/components/LoadingSkeletonThread.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+	<div>
+		<div v-for="i in numberOfLines" :key="i" class="item-list__entry">
```

Comment:
this is not an item list, is it?

Reply:
yes it is a list

Reply:
there is exactly one "item"

Reply:
ah sorry, we are in threads, no its not a list in threads.